backend/src/main/java/ba/unsa/etf/si/bbqms/ws/models/AddTenantDto.java

```diff
@@ -0,0 +1,51 @@
+package ba.unsa.etf.si.bbqms.ws.models;
+
+// note the userId field which I suspect will be necessary in the future
```

Comment:
I kept that field so that the user can get a tenantId when they add a new tenant but I realize this could be accomplished differently. Removed for now.